code/modules/clothing/suits/armor.dm

```diff
@@ -58,7 +59,6 @@
 	desc = "An armored vest that protects against some damage."
 	icon_state = "armorsec"
 	item_state = "armorsec"
-	slowdown = SLOWDOWN_ARMOR_MEDIUM //prevents powergaming marine by swapping armor.
```

Comment:
I'd rather not have more zero slowdown armour, as it is obtainable in normal gameplay and people will inevitiably powergame with it, chest armour only or no.